frontends/main/src/app/page.tsx

```diff
@@ -21,26 +16,8 @@ export async function generateMetadata({
   })
 }
 
-const Page: React.FC = async () => {
-  const queryClient = getQueryClient()
-
-  /*
-   * Prefetch the first tab (All) of the carousel
-   */
-  await queryClient.prefetchQuery(
```

Comment:
Getting rid of this because with a nonzero staletime, the featured course bookmarks are problematic: the resources **are** user-dependent, so they can't really be fetched on the server, unless they are immediately refetched, which we don't want.

We can re-enable this once https://github.com/mitodl/hq/issues/5159 is done